### ISSUE TITLE
Hardcoded bash path?

### ISSUE THREAD

**Issue by julian-hoch** [matejak/argbash#185]
This is just a quesiton - is there a reason why the bash code is hardcoded to /bin/bash?
Since I run NixOS I always have to manually change it o /usr/bin/env bash, since /bin/bash does not exist. But perhaps I am missing something?

**Comment by gdevenyi** [comment]
POSIX standard requires env to be present, so switching to using it for all shebangs is a reasonable change

**Comment by kirtfitzpatrick** [comment]
Agreed. MacOS ships with a very old bash and so anyone on that platform that scripts with bash will invariably use the bash installed via Homebrew in `/opt/homebrew/bin`. I globally replaced all instances of `#!/bin/bash` with `#!/usr/bin/env bash` and ran make and everything worked just fine. Seems like an easy win if it also works for other OS's.

**Comment by matejak** [comment]
Fixed by #187. There is no `/bin/bash` in the source.
